src/logger/HttpLogger_test.go

```diff
@@ -0,0 +1,91 @@
+package logger
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//Testing HTTP Logger
+
+func TestCreateInstance(t *testing.T) {
+
+	//Creating a single instance
+	httpLogger := NewHttpLogger()
+	assert.NotNil(t, httpLogger)
+	assert.Equal(t, HttpLogger.agent, httpLogger.Agent())
+	assert.False(t, httpLogger.Enablable())
+	assert.False(t, httpLogger.Enabled())
+	assert.Nil(t, httpLogger.Queue())
+	assert.Nil(t, httpLogger.Url())
+
+}
+
+func TestCreateMultipleInstances(t *testing.T) {
+	//Creating multiple instances
+	url1 := "https://resurface.io"
+	url2 := "https://whatever.com"
+	helper := GetTestHelper()
+	options1 := Options{
+		url := url1
+	}
+	options2 := Options{
+		url := url2
+	}
+	options3 := Options{
+		url := helper.demoURL
+	}
+
+	logger1 := NewHttpLogger(options1)
+	logger2 := NewHttpLogger(options2)
+	logger3 := NewHttpLogger(options3)
+
+	//Logger 1
+	assert.NotNil(t, logger1)
+	assert.Equal(t, HttpLogger.agent, logger1.AGENT())
+	assert.True(t, logger1.Enablable())
+	assert.True(t, logger1.Enabled())
+	assert.Equal(t, url1, logger1.Url())
+
+	//Logger 2
+	assert.NotNil(t, logger2)
+	assert.Equal(t, HttpLogger.agent, logger2.Agent())
+	assert.True(t, logger2.Enablable())
+	assert.True(t, logger2.Enabled())
+	assert.Equal(t, url2, logger2.Url())
+
+	//Logger 3
+	assert.NotNil(t, logger3)
+	assert.Equal(t, HttpLogger.agent, logger3.Agent())
+	assert.True(t, logger3.Enablable())
+	assert.True(t, logger3.Enabled())
+	assert.Equal(t, helper.demoURL, logger3.Url())
+
+	//Testing Usage Logger
+	//Disable
+	UsageLoggers.disable()
```

Comment:
`UsageLoggers`'s enabled and disabled should be public so 
`UsageLoggers.Disable()`
and `UsageLoggers.Enable()`